# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: services/common/rest.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c services/common/rest.js
--- a/services/common/rest.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/services/common/rest.js	Tue Feb 12 16:08:25 2019 +0000
@@ -326,7 +326,7 @@
 
     // Blast off!
     try {
-      channel.asyncOpen2(this);
+      channel.asyncOpen(this);
     } catch (ex) {
       // asyncOpen can throw in a bunch of cases -- e.g., a forbidden port.
       this._log.warn("Caught an error in asyncOpen", ex);